cmd/lookoutd/common.go

```diff
@@ -84,6 +84,9 @@ type queueConsumerCommand struct {
 
 var defaultInstallationsSyncInterval = 5 * time.Minute
 
+// DiskcachePath is the path used on disk for caching responses
+var DiskcachePath = "/tmp/github"
+
```

Comment:
any reason to make this variable public?

Reply:
no reason 👍